coral/src/app/features/approvals/acls/AclApprovals.tsx

```diff
@@ -421,6 +151,29 @@ function AclApprovals() {
     (request) => request.req_no === Number(detailsModal.reqNo)
   );
 
+  function handleViewRequest(reqNo: string): void {
+    setDetailsModal({ isOpen: true, reqNo });
+  }
+
+  function handleApproveRequest(reqNo: string): void {
+    approveRequest({
+      requestEntityType: "ACL",
+      reqIds: [reqNo],
+    });
+  }
+
+  function handleDeclineRequest(reqNo: string): void {
+    setDeclineModal({ isOpen: true, reqNo });
+  }
+
+  function handleIsBeingApproved(reqNo: string): boolean {
+    return approveVariables?.reqIds[0] === reqNo;
```

Comment:
This will probably not work in the future. The reason why the `reqId`s are in an array is for the upcoming implementation of batch approve/reject. So I think it's probably a little more appropriate to use `approveVariables.includes` or similar approach to handle this ^^

Reply:
> The reason why the reqIds are in an array is for the upcoming implementation of batch approve/reject


Thanks! In that case, definitely :)